src/App.js

```diff
@@ -125,7 +121,7 @@ export default function App() {
               />
 
               <Route path="/about" component={About} />
-              <Route path="/resources" component={Resources} />
+              <Route path="/products" component={Products} />
```

Comment:
We need to support the `/resources` page as well in case we have links pointing to that or if users have bookmarks

```tsx
<Route path="/resources">
  <Redirect to="/products" />
</Route>
```